src/__tests__/pages/guardian-directory.test.tsx

```diff
@@ -96,7 +96,9 @@ describe('GuardianDirectory', () => {
       })
     })
 
-    test('has no a11y violations', async () => {
+    // TODO: There is an a11y violation in the Search component we are importing from the
+    // react-uswds library. We will re-enable this test once the violation is fixed there.
```

Comment:
Thanks for flagging this! Just to connect the dots here's the link to the story @jcbcapps added for this https://app.shortcut.com/orbit-truss/story/2980/fix-a11y-violation-in-search-component